js/util/lru_cache.js

```diff
@@ -111,6 +111,23 @@ class LRUCache<T> {
     }
 
     /**
+     * Remove a key/value combination from the cache.
+     *
+     * @param {string} key the key for the pair to delete
+     * @returns {LRUCache} this cache
+     * @private
+     */
+    remove(key: string) {
+        if (!this.has(key)) { return this; }
+
+        this.onRemove(this.data[key]);
```

Comment:
`cache.get` removes an item from the cache and returns it, so there is some repeated code here, and we _could_ use `cache.get` to delete tiles from the cache, but then it's returning expiring tiles to nowhere and would be less intuitive in code (when cached tile's timer expires, retrieve it from the cache … vs when cached tile's timer expires, remove it from the cache). 😕